crashlogger.js

```diff
@@ -50,24 +49,34 @@ module.exports = function crashLogger(err, description, data = null) {
 
 	if (Config.crashguardemail && ((datenow - lastCrashLog) > CRASH_EMAIL_THROTTLE)) {
 		lastCrashLog = datenow;
-		try {
-			if (!transport) transport = require('nodemailer').createTransport(Config.crashguardemail.options);
-		} catch (e) {
-			console.error(`Could not start nodemailer - try \`npm install\` if you want to use it`);
-		}
+
+		let text = `${description} crashed `;
 		if (transport) {
-			transport.sendMail({
-				from: Config.crashguardemail.from,
-				to: Config.crashguardemail.to,
-				subject: Config.crashguardemail.subject,
-				text: `${description} crashed ${hadException ? 'again ' : ''}with this stack trace:\n${stack}`,
-			}, /** @param {?Error} err */ err => {
-				if (err) console.error(`Error sending email: ${err}`);
-			});
+			text += `again with this stack trace:\n${stack}`;
+		} else {
+			try {
+				transport = require('nodemailer').createTransport(Config.crashguardemail.options);
+			} catch (e) {
+				if (e.code !== 'MODULE_NOT_FOUND') throw e; // Config.crashguardemail was likely misconfigured
+				throw new Error(
```

Comment:
No. When the errors are thrown in the catch statement here, they don't cause any 'uncaughtException' event to be emitted by `process`, so the errors don't get passed back to `crashlogger` again. I'll still need to rewrite this part if `nodemailer` shouldn't be installed by default though